README.md

```diff
@@ -42,7 +58,39 @@ const { Client, GetRecommendations } = require("amazon-connect-wisdomjs");
 import { Client, GetRecommendations } from "amazon-connect-wisdomjs";
 ```
 
-### Usage
+# Initialization
+
+Initializing the WisdomJS client is the fist step to verify that you have everything setup correctly.
+
+```js
+import { Client } from "amazon-connect-wisdomjs";
+
+const wisdomClient = new Client({
+  instanceUrl: "https://your-connect-instance.my.connect.aws",   // required
+  endpoint: "https://your-connect-instance.my.connect.aws/api",  // optional, defaults to '<instanceUrl>'
+  callSource: "agent-app",                                       // optional, defaults to 'agent-app'
+  serviceId: 'Wisdom',                                           // optional, defaults to 'Wisdom'
+  maxAttempts: 3,                                                // optional, defaults to 3
+  logger: {},                                                    // optional, if provided overrides default logger
+  headers: {},                                                   // optional, if provided overrides request headers
+  requestHandler: {},                                            // optional, if provided overrides the default request handler
+});
+```
+
+Initializing the WisdomJS client integrates with Connect by loading the pre-built Wisdom widget located at `<instanceUrl>/wisdom-v2` into an iframe and placing it into a container div. API requests are funneled through this widget and made available to your JS client code.
```

Comment:
This reads strangely.  maybe drop 'Initializing' and just use 'The WIsdomJS client integrates...'

Reply:
will do.